src/styles/blocks/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+  
+  &__title {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+  
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+  
+  &__button {
+    box-sizing: border-box;
+    display: block;
+    height: 40px;
+  
+    background: #00acdc;
+    border-radius: 5px;
+  
+    color: #fff;
+    line-height: 40px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+  
+    &:hover {
+      color: #00acdc;
+      background: #fff;
+      border: 1px solid #00acdc;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover